share/spack/bootstrap/github-actions-v0.2/patchelf.json

```diff
@@ -0,0 +1,34 @@
+{
+  "verified": [
```

Comment:
Can you setup a:
```
share/spack/bootstrap/github-actions-v0.3
```
folder and move `patchelf.json in there + symlink `clingo.json` and `gnupg.json`?

Reply:
I don't see the point of having `v0.1`, `v0.2`, `v0.3`, ... Can't I just rename the directory to 0.3?

Reply:
Ah, it's referenced in the config... let's leave it for compat then. But at some point we could prune those guys...